autoPyTorch/data/tabular_target_validator.py

```diff
@@ -280,7 +297,6 @@ def _check_data(self, y: SupportedTargetTypes) -> None:
                                   )
         if self.type_of_target not in supported_output_types:
             raise ValueError("Provided targets are not supported by AutoPyTorch. "
-                             "Provided type is {} whereas supported types are {}.".format(
-                                 self.type_of_target,
-                                 supported_output_types
-                             ))
+                             "Provided type is {} whereas supported types are {}.".format(self.type_of_target,
```

Comment:
Please refer to [this](https://github.com/automl/Auto-PyTorch/pull/434/files#r895949345) comment